client/index.html

```diff
@@ -345,6 +345,13 @@ <h2>Browser Notifications</h2>
 								</label>
 							</div>
 
+							<div class="col-sm-12">
+								<label class="opt">
+									<label for="highlightsIgnore" class="sr-only">Custom ignore highlights (comma-separated keywords)</label>
+									<input type="text" id="highlightsIgnore" name="highlightsIgnore" class="input" placeholder="Custom ignore highlights (comma-separated keywords)">
```

Comment:
Remove "Custom", it's just about ignoring highlights. There's no "standard" ignoring of highlights (custom highlights is custom because there's standard highlights).